build.sh

```diff
@@ -23,30 +36,30 @@ echo ""
 echo "Which signing identity do you want to use?"
 echo "1. SURFnet B.V. (ZYJ4TZX4UU)"
 echo "2. Egeniq (E85CT7ZDJC)"
-echo "3. Enter own Team ID: "
+echo "3. Enter own Team: "
 read -p "1-3?" choice
 
 
-# Enter custom Team ID.
-
+# Enter custom Team ID and Team Name.
 if [ "$choice" == 3  ]
 then
 read -p "Enter Team ID: " CUSTOMTEAMID
+read -p "Enter Team Name: " CUSTOMTEAMNAME
 fi
 
-#Simple TeamID Validation. Apple Team ID always consists of 10 Character
-
+#Simple TeamID Validation. Apple Team ID always consists of 10 Characters
 if  ! [ "${#CUSTOMTEAMID}" == 10  ]
 then
 echo "Error: Team ID is not valid"
+exit 0
```

Comment:
Typically you exit with a non-zero exit code to indicate failure.

Reply:
HI François Boss,
Ok. I will put 1 here